zola/sass/graph.scss

```diff
@@ -47,4 +47,4 @@ div.docs-graph {
 	width: 100%;
 	// height: 30vh;
 	aspect-ratio: 2;
-}
```

Comment:
here

Reply:
why newline is needed?

Reply:
https://gist.github.com/OleksiyRudenko/d51388345ea55767b7672307fe35adf3